pkg/helper/cardinality/cardinality.json

```diff
@@ -50,7 +50,20 @@
   "DnsErrno": "fine",
   "TimeFlowRttNs": "avoid",
   "NetworkEvents": "avoid",
+  "NetworkEvents>Feature": "fine",
+  "NetworkEvents>Type": "fine",
+  "NetworkEvents>Namespace": "fine",
+  "NetworkEvents>Name": "fine",
+  "NetworkEvents>Action": "fine",
+  "NetworkEvents>Direction": "fine",
   "K8S_ClusterName": "fine",
+  "Sampling": "fine",
+  "ZoneId": "avoid",
+  "XlatSrcPort": "careful",
+  "XlatSrcAddr": "avoid",
+  "XlatIcmpId": "avoid",
+  "XlatDstPort": "careful",
+  "XlatDstAddr": "avoid",
```

Comment:
@jotak changes here LGTM, I am thinking to verify this based on code review, let me know if you think e2e test for the feature is required.

Reply:
Thanks @memodi - I think it's fine as is - I just added an integration test (https://github.com/netobserv/network-observability-operator/pull/999/commits/c8f3f11b38101d0196dbae8b4cddc00d41a6e774) to add on verification